e2e-tests/tests/init-deploy/00-deploy-operator.yaml

```diff
@@ -4,10 +4,10 @@ metadata:
   name: init-deploy-resource-quota
 spec:
   hard:
-    requests.cpu: "10"
-    requests.memory: "16Gi"
-    limits.cpu: "20"
-    limits.memory: "32Gi"
+    requests.cpu: "2"
```

Comment:
Numbers according to : https://docs.percona.com/percona-operator-for-mysql/pxc/System-Requirements.html#resource-limits